keystrokes/src/components/Character/CharacterDetails/CharacterKeybindDisplay.js

```diff
@@ -9,16 +9,62 @@ import {
 	ExpansionPanelSummary,
 	ExpansionPanelDetails,
 } from '@material-ui/core';
+import ReactQuill from 'react-quill';
+import 'react-quill/dist/quill.snow.css';
+import ReactHtmlParser from 'react-html-parser';
+import update from 'immutability-helper';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import styleGuide from '../../../stylesheets/style';
 import { TabPanel } from '../helpers/TabPanels';
 
 CharacterKeybindDisplay.propTypes = {
 	index: PropTypes.number,
+	character: PropTypes.object,
+	setCharacter: PropTypes.func,
+	spec: PropTypes.number,
 	keyBinding: PropTypes.number,
 };
-function CharacterKeybindDisplay({ keyBinding, index }) {
+function CharacterKeybindDisplay({
+	keyBinding,
+	spec,
+	index,
+	character,
+	setCharacter,
+}) {
 	const classes = styleGuide();
+	const [descriptionToggle, setDescriptionToggle] = useState(false);
+	const [descriptionText, setDescriptionText] = useState(
+		character.specs[spec].keybindings[keyBinding][
+			Object.keys(character.specs[spec].keybindings[keyBinding])[0]
+		].description || ''
+	);
+
+	function toggleDescription(saving = false) {
+		if (saving) {
+			const updatedCharacter = update(character, {
```

Comment:
Nice usage. 
I think the final object could be moved out to another line to make it not soo deep and maybe a bit easier to read but looks good as far as i can see